documentation/docs/ui/global_customizations/chat_ui_customizations/navigator.md

```diff
@@ -0,0 +1,31 @@
+---
+id: uiGlobalCustomizationsNavigator
+title: Navigator
+sidebar_position: 2
+---
+
+It is possible to add a custom navigator to `ChatUI` by adding a new `ChatNavigator`.
+
+A user can instantiate a `ChatNavigator` and provide its custom implementation of `ChatNavigationHandler`:
```

Comment:
Would be nice to explain when we are using the navigator(AttachmenDestination, WebDestiation)